app/views/pages/attend.html.erb

```diff
@@ -1 +1,2 @@
 <h1>Attend</h1>
+<%= render './shared/top_banner' %>
```

Comment:
please also update all other render statements in the views (maybe for unrelated views create a seperate PR for it)